Framework/script/RepoCleaner/qcrepocleaner/rules/multiple_per_run.py

```diff
@@ -85,7 +97,29 @@ def process(ccdb: Ccdb, object_path: str, delay: int,  from_timestamp: int, to_t
             logger.debug(f"     not in the allowed period, skip this bucket")
             preservation_list.extend(run_versions)
         else:
-            logger.debug(f"     not in the grace period")
+            logger.debug(f"    not in the grace period")
+
+            if delete_first_last:
```

Comment:
I don't quite understand why we only need the flag for the 2nd object since SOR, but not the second from the last. Won't we delete the last object each time we run the policy again?

Reply:
I don't think so because we work on the run as a whole. If we delete the first then we delete the last, if we don't delete the first we don't delete the last. 
The mark on the second is a mark we use for the whole run.

Reply:
ah, ok, clear